RobotResource final_resource_state
---
float32 cost_to_complete